src/Components/v2/CarouselV2.tsx

```diff
@@ -0,0 +1,241 @@
+import { Box, ChevronIcon, Flex } from "@artsy/palette"
+import React, { ReactNode, useEffect, useRef, useState } from "react"
+import styled from "styled-components"
+import { left, LeftProps, right, RightProps } from "styled-system"
+import { Media } from "Utils/Responsive"
+
+type Arrow = (
+  props: {
+    Arrow: React.ReactType
+    getFlickity: any
+  } & Props
+) => React.ReactNode
+
+interface Props {
+  height?: number
+  data: object[] // This is designed to handle any shape of data passed, as long as its an array
+  render: (slide) => ReactNode
+  renderLeftArrow?: Arrow
+  renderRightArrow?: Arrow
+  onArrowClick?: () => void
+  settings?: Flickity.Options
```

Comment:
`import { Options as FlickityOptions } from 'flickity'`

Reply:
Also, since `flickity` uses "Options" language, might be worth renaming `settings` to `options` so that things are 1:1.